azurerm/provider.go

```diff
@@ -116,6 +116,7 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_builtin_role_definition":                dataSourceArmBuiltInRoleDefinition(),
 			"azurerm_cdn_profile":                            dataSourceArmCdnProfile(),
 			"azurerm_client_config":                          dataSourceArmClientConfig(),
+			"azurerm_container_orchestrator_versions":        dataSourceArmContainerOrchestratorVersions(),
```

Comment:
as discussed IRL, can we make this:

```suggestion
			"azurerm_kubernetes_service_versions":        dataSourceArmKubernetesServiceVersions(),
```